README.md

```diff
@@ -23,11 +24,36 @@ vim .env
 ### Running
 ```bash
 set -a && source .env
+python start.py -r
+# or
 python -m app.main
 # or
 uvicorn --reload app.main:app --log-level=info
 ```
 
+It is assumed that you've configured your .env properly. This includes the expectation that a Postgres
```

Comment:
I think we should develop a docker-compose for local development soon that stands all these up as containers at once. What do you think @frostyfan109?

Reply:
Yes, that's a good idea, I agree. I'll capture it in a ticket.